### PR TITLE
Check-in #30729 - Add last updated message & refresh link

### PR DESCRIPTION
**Pull request by acrollet**
## Description

This PR allows Veterans to see when their appointment was last loaded, and fetch the latest data.

Sketch: https://www.sketch.com/s/e79a827e-42cf-4a82-b554-874c75b5c70e/a/9PAAw1z

## Original issue(s)

department-of-veterans-affairs/va.gov-team#30729

## Screenshots

Screen Shot 2021-10-07 at 11 53 51 AM
Screen Shot 2021-10-07 at 11 53 02 AM
Screen Shot 2021-10-07 at 11 52 18 AM
Screen Shot 2021-10-07 at 11 51 01 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by acrollet** [comment]
This still needs tests, posting as a draft for now.

File: src/applications/check-in/pages/CheckIn/DisplayMultipleAppointments.jsx [hunk 1]
```diff
@@ -44,6 +50,15 @@ export default function DisplayMultipleAppointments(props) {
           );
         })}
       </ol>
+      <p data-testid="update-text">
+        <strong>Latest update:</strong>{' '}
+        {format(new Date(), 'MMMM dd, yyyy HH:mm')}
```
Comment (mdewey) [inline src/applications/check-in/pages/CheckIn/DisplayMultipleAppointments.jsx:55]: Does this also refresh when the button is clicked?

Comment (acrollet) [reply src/applications/check-in/pages/CheckIn/DisplayMultipleAppointments.jsx:55]: yep! 😃

**Comment by acrollet** [comment]
@kristenmcconnell I've pushed the change and updated the screenshot in the description.

**Comment by kristenmcconnell** [comment]
One more small thing, which I apologize for not being clearer in my description. Time is 00:00 or 0:00, so it would be 9:23 not 09:23.

File: src/applications/check-in/tests/e2e/phase-3-multiple-appointments/multiple-appointments/veterans.may.refresh.their.appointments.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      rv2.payload.appointments = [newLatest, earliest, midday];
+      const responses = [rv1, rv2];
+
+      cy.intercept(
+        {
+          method: 'GET',
+          url: '/check_in/v2/patient_check_ins/*',
+        },
+        req => {
+          req.reply(responses.shift());
```
Comment (mdewey) [inline src/applications/check-in/tests/e2e/phase-3-multiple-appointments/multiple-appointments/veterans.may.refresh.their.appointments.cypress.spec.js:45]: That's a clever use of the shift. I like it.

**Review by mdewey — APPROVED** [review]
LGTM. All tests pass locally, Great work

**Comment by kristenmcconnell** [comment]
Looks good to me now. @ytsaoca Do you see anything else? 

I do have a question about the "Check in again" link. Is that just for engineering testing purposes? It won't be part of the release, correct? @mdewey

**Comment by acrollet** [comment]
@kristenmcconnell that's right, I only added it to the multiple appts page :-)

**Pull request merged**